codecarbon/emissions_tracker.py

```diff
@@ -28,6 +29,19 @@
 )
 logger = logging.getLogger(__name__)
 
+# /!\ Warning: current implementation prevents the user from setting any value to None
+# from the script call
+# Imagine:
+#   1/ emissions_endpoint=localhost:8000 in ~/.codecarbon.config
+#   2/ Inside the script, the user cannot disable emissions_endpoint with
+#   EmissionsTracker(emissions_endpoint=None) since the config logic will use the one in
+#   the config file.
+#
+# To fix this, a complex move would be to have default values set to the sentinel:
+# _sentinel = object()
+# see: https://stackoverflow.com/questions/67202314/
+#      python-distinguish-default-argument-and-argument-provided-with-default-value
```

Comment:
That could be problematic in some cases

Reply:
How is it handled in comet? Is there a sentinel, something else or nothing at all?

It looks like it's not handled but I could be wrong